src/AutomationTests/OutputFileHelperUnitTests.cs

```diff
@@ -157,7 +155,7 @@ public void OutputFileHelperCtor_WithOutputDirectory_UsesProvidedDirectory()
 
         [TestMethod]
         [Timeout(1000)]
-        public void OutputFileHelperCtor_CreatesNonexistentDirectory()
+        public void OutputFileHelper_EnsureOutputDirectoryExists_CreatesNonexistentDirectory()
```

Comment:
It feels like we're only covering 2 of 3 cases. We validate the exception and if the directory doesn't exist, but we don't cover the case where directory already exists (and CreateDirectory never gets called)